File: VB/T457705/Form1.vb
Imports DevExpress.XtraReports.UI

Namespace T457705
	Partial Public Class Form1
		Inherits DevExpress.XtraEditors.XtraForm

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			Dim report As New XtraReport1()
			Dim tool As New ReportPrintTool(report)
			tool.ShowRibbonPreviewDialog()
		End Sub
	End Class
End Namespace

File: CS/T457705/Form1.cs
using System;
using DevExpress.XtraReports.UI;

namespace T457705 {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            var report = new XtraReport1();
            ReportPrintTool tool = new ReportPrintTool(report);
            tool.ShowRibbonPreviewDialog();
        }
    }
}
